common/src/main/java/com/microsoft/identity/common/internal/request/OperationParameters.java

```diff
@@ -293,5 +321,23 @@ public void validate() throws ArgumentException {
             }
         }
 
+        // AuthenticationScheme is present...
+        if (null == mAuthenticationScheme) {
```

Comment:
This would be `null` though in the case of `Bearer`, right? If yes, then is this check needed?

Reply:
So I saw that you were setting it in the `TokenParameters` in MSAL. I think it may be better to just set this in the `OperationParametersAdapter`

Reply:
For calls to either `acquireToken()` or `acquireTokenSilent()` this value shouldn't be `null` after passing through the [`TokenParameters.Builder`](https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/iambmelt/at-pop-sandbox/msal/src/main/java/com/microsoft/identity/client/TokenParameters.java#L52-L59). If this was guaranteed to be the only place we ever create `OperationParams` I would say let's remove this validation. I'll leave it here for now unless there's some compelling reason to remove it?